### DISCUSSION TITLE
Board scrolling broken on mobile

### DISCUSSION THREAD

**Discussion by JVimes** [community/community#54009] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

When I try to scroll a project board on mobile, it always grabs a card and moves it as well as scrolling.

It seems impossible to use without accidentally moving cards.

Anyone else seeing this? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by FinnurO** [comment]
Happening to me as well
